direct/common/subsample.py

```diff
@@ -324,6 +404,90 @@ def mask_func(
         return torch.from_numpy(self._reshape_and_broadcast_mask(shape, mask))
 
 
+class CartesianEquispacedMaskFunc(FastMRIEquispacedMaskFunc):
+    r"""Cartesian equispaced vertical line mask function.
+
+    Similar to :class:`FastMRIEquispacedMaskFunc, but instead of center fraction (`center_fractions`) representing
+    the fraction of center lines to the original size, here, it represents the actual number of center lines.
+    """
```

Comment:
Perhaps you can sort the classes alphabetically just like you do the export in __init__.py

Reply:
Not possible, `FastMRIEquispacedMaskFunc` for example, is parent of `CartesianEquispacedMaskFunc`